manticore/platforms/evm.py

```diff
@@ -1921,12 +1933,26 @@ def tx_gasprice(self):
     def depth(self):
         return len(self._callstack)
 
-    def new_address(self):
+    def new_address(self, sender=None, nonce=None):
```

Comment:
What if we get rid of nonce as an argument? A quick check did not get me any calls to this method with `nonce != None` (except from itself maybe)

Reply:
A human in a script might call it, and they could choose to use `nonce=None` as a convenience.